atc/api/configserver/save.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/tedsuo/rata"
 )
 
+const (
+	JsonContent = "application/json"
```

Comment:
HTTP headers are pretty well defined (i.e. does not change) and understood across languages and platforms. I think it's easier to understand the direct literals in this case than references.

The headers also appear across the system in at least 40 more files. To maintain consistency and hence readability all those would have to be changed as well, which would be a pretty big diff without adding much in the way of readability.

It would be better to retract the header related changes.